doc/source/whatsnew/v0.21.0.txt

```diff
@@ -180,7 +180,7 @@ Groupby/Resample/Rolling
 
 Sparse
 ^^^^^^
-
+- Bug in SparseSeries raises AttributeError when a dictionary is passed in as data (:issue:`16777`)
```

Comment:
Could you add double ticks ("``") around both `SparseSeries` and `AttributeError`?